Hedvig/Screens/ProfileScreen/Viewables/MyPaymentRow.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  MyInfoRow.swift
+//  Hedvig
+//
+//  Created by Sam Pettersson on 2019-01-03.
+//  Copyright © 2019 Hedvig AB. All rights reserved.
+//
+
+import Flow
+import Form
+import Foundation
+import Presentation
+
+struct MyPaymentRow {
+    let monthlyCost: Int
+    let presentingViewController: UIViewController
+}
+
+extension MyPaymentRow: Viewable {
+    func materialize(events: SelectableViewableEvents) -> (IconRow, Disposable) {
+        let bag = DisposeBag()
+
+        let row = IconRow(
+            title: String.translation(.PROFILE_PAYMENT_ROW_HEADER),
+            subtitle: "\(monthlyCost) kr/XYZ • \(String.translation(.PROFILE_MY_PAYMENT_METHOD))",
```

Comment:
we should fix support for replacement translations 🤔

Reply:
@Iamsamwhoami I agree, we should fix it. Though i'm unsure how to do so in this project :/